### PR TITLE
[CNTO-3426]: figma icons added

### PR DESCRIPTION
**Pull request by Indra-1996**

### CODE REVIEW

File: src/components/Icon/index.js [hunk 1]
```diff
@@ -22,6 +28,30 @@ const Loading = ({
...
+  'active-conversation': activeConversation,
+  'add-notes': addNotes,
+}
+
+const Custom = ({ type, message }) => {
+  if (typeToImageMapping[type]) {
+    return (
+      <div className="sg contacto-image-wrapper">
+        <div className="contacto-image">
+          <img src={typeToImageMapping[type]} alt="style-guide-text.png" />
```
Comment (samuellawerentz) [inline src/components/Icon/index.js:45]: Can we update the alt text to the message? or some other meaningful text.

**Review by samuellawerentz — CHANGES_REQUESTED** [review]
Great work!

File: src/components/Icon/index.js (hunk 1 above)
Comment (Indra-1996) [reply src/components/Icon/index.js]: sure

**Review by samuellawerentz — APPROVED** [review]
@Indra-1996  Please check with @malay2  he is releasing some version. We can club the releases together.

**Pull request closed without merging**